src/faldbt/project.py

```diff
@@ -436,6 +487,15 @@ def _get_models_with_keyword(self, keyword) -> List[DbtModel]:
             filter(lambda model: keyword in model.meta, self._faldbt.list_models())
         )
 
+    def _map_tests_to_models(self, models: List[DbtModel], tests: List[DbtTest]) -> List[DbtModel]:
+        for model in models:
+            for test in tests:
+                if test.model == model.name:
+                    model.tests.append(test)
+                    if test.status != 'skipped':
+                        model.set_status('tested')
```

Comment:
You are overwritting model status with test status several times

Reply:
Thanks! This should be resolved now.